### DISCUSSION TITLE
Is it possible to use WGPU 22.x with Winit 0.29.x?

### DISCUSSION THREAD

**Discussion by John-Nagle** [gfx-rs/wgpu#6411] [Q&A] [answered]
Is it possible to use WGPU 22.x with Winit 0.29.x? I'd like to debug the churn in WGPU separately from the churn in Winit.

**Comment by cwfitzgerald** [comment, marked as answer]
Yes. 0.29.15 has a default-enabled [rwh_06](https://docs.rs/crate/winit/0.29.15/features) feature which enables raw window handle 0.6, which is the same as used by wgpu 22.x
